instrumentation/opentelemetry-instrumentation-grpc/src/opentelemetry/instrumentation/grpc/_server.py

```diff
@@ -125,18 +126,16 @@ def set_code(self, code):
         self.code = code
         # use details if we already have it, otherwise the status description
         details = self.details or code.value[1]
+        self._active_span.set_attribute("rpc.status_code", code.name)
```

Comment:
is this a semantic convention from the spec? [I don't see an entry there](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/rpc.md#attributes)

Reply:
I don't either, but then, the spec you linked is now out of date, specifically this part:
```
Implementations MUST set status which MUST be the same as the gRPC client/server status. The mapping between gRPC canonical codes and OpenTelemetry status codes is 1:1 as OpenTelemetry canonical codes is just a snapshot of grpc codes which can be found here.
```

Reply:
Ah, see https://github.com/open-telemetry/opentelemetry-specification/issues/1044 - I didn't realize this discussion was going on when I made this change.

I'd be happy to remove the extra attribute until that's resolved, but not having the status code in a trace makes the trace significantly less useful for real use.

Reply:
Ok I complied with your change to that spec :)

Reply:
Cool, I was about to ask this 😄